modules/home/components/InformationParticipateMakerGovernance.tsx

```diff
@@ -0,0 +1,268 @@
+  titleFirst: string;
+  titleSecond: string;
+  image: string;
+  description: React.ReactNode;
+  color: string;
+  links: {
+    linkHref: string;
+    linkTitle: string;
+  }[];
+};
+
+function Card({ infoPoint }: { infoPoint: InfoPoint }): React.ReactElement {
+  return (
+    <Box sx={{ height: '100%', background: ['#ffffffb0', '#ffffffb0', 'transparent'] }}>
+      <Flex sx={{ height: '100%' }}>
+        <Box sx={{ p: 3, display: ['none', 'block'] }}>
+          <Heading as="h1" sx={{ fontSize: '32px' }}>
+            <Text sx={{ color: infoPoint.color }}>{infoPoint.number}</Text>
+          </Heading>
+        </Box>
+        <Flex
+          sx={{
+            flexDirection: 'column',
+            justifyContent: 'space-between',
+            height: '100%',
+            flex: 1,
+            background: `url(${infoPoint.image});`,
+            backgroundPositionX: '100%',
+            backgroundRepeat: 'no-repeat',
+            p: 3,
+            pl: 2,
+            backgroundSize: 'auto 100%'
+          }}
+        >
+          <Box>
+            <Heading as="h1" sx={{ mb: 3 }}>
+              <Text>{infoPoint.titleFirst}</Text>
+              <Text sx={{ color: infoPoint.color, ml: 1 }}>{infoPoint.titleSecond}</Text>
+            </Heading>
+
+            <Box sx={{ width: ['100%', '100%', '50%'] }}>{infoPoint.description}</Box>
+          </Box>
+          <Box>
+            {infoPoint.links.map(link => (
+              <Box mb={2} key={link.linkHref}>
+                <ExternalLink
+                  href={link.linkHref}
+                  title={link.linkTitle}
+                  styles={{ color: 'inherit', fontWeight: 'semiBold' }}
+                >
+                  <Flex sx={{ alignItems: 'center' }}>
+                    <Text>{link.linkTitle}</Text>
+                    <Icon ml={2} name="arrowTopRight" size={3} sx={{ color: infoPoint.color }} />
+                  </Flex>
+                </ExternalLink>
+              </Box>
+            ))}
+          </Box>
+        </Flex>
+      </Flex>
+    </Box>
+  );
+}
+export default function InformationParticipateMakerGovernance(): React.ReactElement {
+  const infoPoints: InfoPoint[] = [
+    {
+      number: '01',
+      title: 'Understand off-chain governance',
+      titleFirst: 'Understand',
+      titleSecond: 'off-chain governance',
+      links: [
+        {
+          linkHref: 'https://manual.makerdao.com/governance/voting-in-makerdao/off-chain-governance',
+          linkTitle: 'Learn more about off-chain governance'
+        }
+      ],
+      image: '/home/understand-governance/first.png',
+      color: '#1AAB9B',
+      description:
+        "Off-chain governance refers to processes for making decisions that don't require on-chain voting and gathering feedback prior to on-chain voting. Off-chain governance happens on the Maker Governance Forum, where the community meets to propose and discuss new proposals. Anyone can participate in off-chain governance."
+    },
```

Comment:
Just a suggestion but maybe all of these can be in a separate util file instead of in the component. Makes the component code less cluttered. I did something similar for the resources section. Wdyt?

Reply:
maybe i can create a folder for the component and put the data inside